helm-charts/chatqna/gaudi-values.yaml

```
@@ -13,9 +13,15 @@ tgi:
   resources:
     limits:
       habana.ai/gaudi: 1
-  MAX_INPUT_LENGTH: "1024"
-  MAX_TOTAL_TOKENS: "2048"
+  MAX_INPUT_LENGTH: "2048"
+  MAX_TOTAL_TOKENS: "4096"
```

This could explain need to increase the limits from default:
```suggestion
  # higher limits are needed with extra input tokens added by rerank
  MAX_INPUT_LENGTH: "2048"
  MAX_TOTAL_TOKENS: "4096"
```